base/dump-dotnet2/Function.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Amazon.Lambda.Core;
+using Amazon.Lambda.S3Events;
+using Amazon.S3;
+using Amazon.S3.Util;
```

Comment:
The following packages, in red, are not being used.

```diff
using System;
-using System.Collections.Generic;
using System.Diagnostics;
-using System.IO;
-using System.Linq;
using System.Threading.Tasks;

using Amazon.Lambda.Core;
-using Amazon.Lambda.S3Events;
using Amazon.S3;
-using Amazon.S3.Util;
```

`using System.IO;` should be left in if the method signature down later in the file is correct and `Stream` is actualy needed, but it doesn't look the variable `stream` is being used anywhere.

```csharp
public async Task<string> FunctionHandler(Stream stream)
{
Console.WriteLine ("Execution started for Function {0} with {1}", context.FunctionName, evnt);
    var environment =  Environment.GetEnvironmentVariables();
    foreach(var env in environment.Keys){
            Console.WriteLine($"{env}:{environment[env]}");
    }

    string filename = "dotnet2.tgz";
    string cmd = $"tar -cpzf /tmp/{filename} --numeric-owner --ignore-failed-read /var/runtime /var/lang";

    var output = cmd.Bash();

    Console.WriteLine("Zipping done! Uploading...");
    await S3Client.PutObjectAsync(new Amazon.S3.Model.PutObjectRequest{
        BucketName="wellthen",
        Key=filename,
        FilePath=$"/tmp/{filename}"
    });
    return "OK";

}
```